test/mitmproxy/addons/test_intercept.py

```diff
@@ -89,3 +89,17 @@ async def test_udp():
         f = tflow.tudpflow()
         await tctx.cycle(r, f)
         assert not f.intercepted
+
+
+async def test_websocket_message():
+    r = intercept.Intercept()
+    with taddons.context(r) as tctx:
+        tctx.configure(r, intercept='~b "hello binary"')
```

Comment:
Is there any better way to filter out websocket messages alone?
`~websocket` intercepts the response of the websocket upgrade request before intercepting websocket messages

Reply:
Not yet - I guess we should consider adding something like `~wq` and `~ws` that works analogous to `~q` and `~s` for WebSocket messages from the client and server.

For the test here this is fine. :)

Reply:
I've added these in my GSoC proposal ;)